pycheribuild/projects/cross/opensbi.py

```diff
@@ -201,3 +215,43 @@ def run_tests(self):
             give_tty_control=True,
             cwd="/",
         )
+
+
+class BuildAllianceOpenSBI(BuildOpenSBI):
+    target = "cheri-alliance-opensbi"
+    _default_install_dir_fn = ComputedDefaultValue(
+        function=lambda config, p: config.cheri_alliance_sdk_dir / "opensbi/riscv64",
+        as_string="$SDK_ROOT/cheri-alliance-opensbi/riscv64"
+    )
+    repository = GitRepository("https://github.com/CHERI-Alliance/opensbi",
+                               default_branch="codasip-cheri-riscv")
+    supported_architectures = (
+        CompilationTargets.FREESTANDING_RISCV64,
+        CompilationTargets.FREESTANDING_RISCV64_PURECAP,
+    )
+
+    def _qemu_install_dir(self) -> Path:
+        return BuildCheriAllianceQEMU.get_install_dir(self, cross_target=CompilationTargets.NATIVE)
+
+    def setup(self):
+        super().setup()
+        self.make_args.set(FW_TEXT_START=0x80000000)
```

Comment:
It's supposed to be 0. It's built as a PIE if possible. The expectation is it self-relocates at startup.